src/Language/PureScript/CoreImp/Module.hs

```diff
@@ -14,6 +15,6 @@ data Module = Module
   , modExports :: [Export]
   }
 
-data Import = Import Text PSString
+data Import = Import (Maybe SourceSpan) Text PSString
```

Comment:
Not to self: the `Maybe` is needed here because `Import` is also used for FFI files. Since we don't parse FFI files, we don't currently have a way to determine what such their source spans are. The `Just` is only used for PureScript module imports